src/app/onboard/name/page.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { useState } from "react";
+
+import UabcHeaderText from "@/components/AuthLogin/UabcHeaderText";
+import { Button } from "@/components/Button";
+import { TextInput } from "@/components/TextInput";
+import { useAccountStore } from "@/stores/accountDetails";
+
+export default function NamePage() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+
+  const updateFirstName = useAccountStore((state) => state.setFirstName);
+  const updateLastName = useAccountStore((state) => state.setLastName);
+
+  return (
+    <div className="h-[100dvh] w-[100dvw]">
+      <div className="mx-4 flex flex-col h-full justify-between">
+        <div className="pt-4">
+          <UabcHeaderText />
+        </div>
+        <div className="flex flex-col space-y-4">
```

Comment:
increase spacing to 24px
```suggestion
        <div className="flex flex-col gap-6">
```